app/services/open_library_service.rb

```diff
@@ -0,0 +1,50 @@
+require 'httparty'
+require 'json'
+
+class OpenLibraryService
+  include HTTParty
+  include Concurrent::Async
+  def initialize(isbn)
+    @isbn = isbn
+    @base_uri = Rails.application.credentials.development[:open_library][:base_uri]
+  end
+
+  def book_info
+    response = book
+    if response.code == 200
+      hash = JSON.parse(response.body).with_indifferent_access
+      build_response(hash)
+    else
+      {
+        code: response.code,
+        message: response.message
+      }
+    end
+  end
+
+  private
+
+  def book
```

Comment:
rename method to `get_book`

Reply:
Rubocop has conflict with that type of name for a method that returns something, it asks us to remove the "get_"

Docs: [https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/AccessorMethodName](url)

## Conflict
![Captura de Pantalla 2021-05-27 a la(s) 12 51 40](https://user-images.githubusercontent.com/52846611/119857868-4e498d80-beea-11eb-944c-934d679c95bf.png)

Reply:
`find_book`, important specify the functionality of methods in the name

Reply:
with that name is accepted